pandas/tests/frame/test_apply.py

```diff
@@ -1367,3 +1367,96 @@ def test_consistency_of_aggregates_of_columns_with_missing_values(self, df, meth
         tm.assert_series_equal(
             none_in_first_column_result, none_in_second_column_result
         )
+
+
+class TestDataFrameNamedAggregate:
+
+    # GH 26513
```

Comment:
usually these go inside the test func

Reply:
yeah, sorry, i put this outside was because i wanted it for all functions in the entire class, since they are all coming from the same issue. I changed and put them under each function.